pinot-query-runtime/src/test/resources/queries/NumericTypes.json

```diff
@@ -0,0 +1,117 @@
+{
+  "smallint": {
+    "psql": "8.1.1",
+    "ignored": true,
+    "comment": "not supported"
+  },
+  "integers": {
+    "comment": "we don't support BIGINT notation",
+    "tables": {
+      "ints": {
+        "schema": [
+          {"name": "int32", "type": "INT"},
+          {"name": "int64", "type": "LONG"}
+        ],
+        "inputs": [
+          [0, 0],
+          [123, 321],
+          [-2147483648, -9223372036854775808],
+          [2147483647, 9223372036854775807]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "psql": "8.1.1",
+        "description": "test selecting integer values",
+        "sql": "SELECT * FROM {ints}"
+      },
+      {
+        "psql": "8.1.1",
+        "ignored": true,
+        "comment": "we don't properly support overflow behavior because we return doubles",
+        "description": "overflow behavior",
+        "sql": "SELECT int32 + 1, int32 - 1, int64 + 1, int64 -1 FROM {ints}",
+        "expect": "*out of range*"
+      }
+    ]
+  },
+  "numeric": {
+    "comment": "ANSI SQL calls this type NUMERIC(precision, scale) but we call it BIG_DECIMAL",
+    "tables": {
+      "numeric": {
+        "schema": [
+          {"name": "big", "type": "BIG_DECIMAL"}
+        ],
+        "inputs": [
+          ["92233720368547758071"],
+          ["92233720368547758071.0000000001"]
+        ]
+      }
+    },
+    "queries": [
+      {
+        "psql": "8.1.2",
+        "TODO": "for some reason, when we select * with this test the timestamp column gets returned as well...",
```

Comment:
if this is true why is the test going through? shouldn't it failed b/c the number of columns don't match H2?

Reply:
correct, that's why I'm not selecting `*` and instead selecting just the column :)

Reply:
oh. I thought you meant select just the column will result the same as select *, my mistake